activitypub.php

```diff
@@ -144,3 +148,49 @@ function( \Friends\Feed $friends_feed ) {
 		$friends_feed->register_parser( Friends_Feed_Parser_ActivityPub::SLUG, new Friends_Feed_Parser_ActivityPub( $friends_feed ) );
 	}
 );
+
+/**
+ * Disable webfinger for known example domains.
+*/
+add_filter(
```

Comment:
shouldn't this be better part of the test and/or of the `debug.php` file?

Reply:
Hm, the problem that I've been running into is that the `the_content` hook is called for any posts. So if you write a post that talks about Mastodon and uses example-mentions, the filter would kick in and try to look it up.

Reply:
Ok, I have decided to move the whole parser into the Friends plugin https://github.com/akirk/friends/pull/137 so that I can add a filter there and I also feel it's better for me to be able to iterate more quickly on it for everything that does not need any ActivityPub protocol work.